src/index.js

```diff
@@ -422,15 +423,15 @@ const _startLogin = (context, event, actionMeta) => (callbackParent, onEvent) =>
       console.log('MESSAGE: ', json)
 
       switch (json.msg) {
-        case 'authorized':
+        case MasqMessages.AUTHORIZED:
           callbackParent({
             type: 'AUTHORIZED',
```

Comment:
Don't you want to reuse messages here too ?

Reply:
'AUTHORIZED' here is the name of the event not the message sent by Masq-app

Reply:
Yeah but they seem highly correlated

Reply:
But not in all cases, one Message does not correspond to one Event so it would be confusing to use the same constant, but we could defined another if needed